vllm/worker/cpu_model_runner.py

```diff
@@ -367,6 +366,8 @@ def execute_model(
             "attn_metadata": model_input.attn_metadata,
             **(model_input.multi_modal_kwargs or {}),
         }
+        if model_input.multi_modal_kwargs:
+            execute_model_kwargs.update(model_input.multi_modal_kwargs)
```

Comment:
This is already done by the unpack operation above.

Reply:
ah thanks! Somehow missed in rebase.